fr/en-cours/originales/transcription-automatisee-graphies-non-latines.md

```diff
@@ -603,30 +605,30 @@ Tableau 4: Exemple de types de projets disponibles et prêts à l'emploi gratuit
 
 <div class="alert alert-warning">
 Le modèle d'analyse de la mise en page semble donc aisément fine-tunable. La reconnaissance de texte, malgré un modèle de grec déjà disponible, s'annonce plus compliquée. Un nouveau choix architectural s'avèrera peut-être pertinent.
 </div>
 
-### Quel volume de données
+#### Quel volume de données
 
 Il est très difficile d'anticiper le nombre de données nécessaire pour le *fine-tuning* des modèles. Une évaluation de la plateforme montre une adaptation pertinente de l'analyse de la mise en page et de la classification des zones de texte dès 50 pages pour des mises en page complexes sur des manuscrits arabes[^35]. Le problème est ici plus simple (moins de variabilité du contenu). Pour la détection des lignes, vingt-cing pages suffisent[^36]. Il n'est toutefois pas nécessaire d'atteindre ces seuils pour mesurer le gain dans l'analyse et la détection.
 
-Au niveau de la transcription, l'état de l'art met en évidence un besoin minimal de 2000 lignes pour entraîner un modèle OCR / HTR[^37], ce qui peut correspondre à une moyenne entre 75 et 100 pages pour des documents manuscrits sur les scripta non latines. Pour la PG, au regard de la densité particulière du texte, cela correspond à une moyenne de 50 pages.
+Au niveau de la transcription, l'état de l'art met en évidence un besoin minimal de 2000 lignes pour entraîner un modèle OCR / HTR[^37], ce qui peut correspondre à une moyenne entre 75 et 100 pages pour des documents manuscrits sur les *scripta* non latines. Pour la PG, au regard de la densité particulière du texte, cela correspond à une moyenne de 50 pages.
 
 <div class="alert alert-warning">
-Ströbel et al. montrent par ailleurs qu'au-delà de 100 pages il n'existe pas de grande différence entre les modèles pour un problème spécifique donné. L'important n'est donc pas de miser sur un gros volume de données, mais au contraire concentrer l'attention sur la qualité des données produites et leur adéquation avec l'objectif recherché.
+Ströbel et al.<a href="#fn:37"><sup>37</sup></a> montrent par ailleurs qu'au-delà de 100 pages il n'existe pas de grande différence entre les modèles pour un problème spécifique donné. L'important n'est donc pas de miser sur un gros volume de données, mais au contraire concentrer l'attention sur la qualité des données produites et leur adéquation avec l'objectif recherché.
```

Comment:
@anisa-hawes this is not really orthodox, I'well aware of it. Is there a way to add new footnotes without having to change the numbering by hand ?

Reply:
Hello @matgille. I'm actually not sure... My preference would be if we can use the usual method, sequentially numbering the notes like this `[^37]`. I realise it is a bit of a pain, and I'm happy to make a note to do this numbering for you when it comes to typesetting the lesson.

Reply:
@matgille @anisa-hawes Do not know if I get the discussion correctly. But if you want to add new notes in a markdown file, irrespectively of the numbering inside, the numerotation of the notes in the html rendering will display correctly. Say, if you add [^200] in the file and in the text it is to be added between notes 30 and 31, you can add it such and it will display correctly in the rendering (if this can save Anisa some time and energy).

Reply:
Hi @anisa-hawes @spapastamkou, 
thank you for your answers ! 

Ok Sofia, so I don't have to worry about the order of the footnotemarks, the important is to link the marks with the corresponding endnote, am I right ? 


@anisa-hawes I will change the files myself, don't worry. 


Best, 


Matthias

Reply:
I believe so @matgille, you can take a look here: https://docs.github.com/en/enterprise-cloud@latest/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes 
And we also have a lesson like this (this is how I learnt about this while editing or publishing), if I manage to find which one, I will let you know

Reply:
Hello @matgille. If it is the case that you want to add an endnote in between `[^37]` and `[^38]` but avoid re-numbering all the notes from that point onwards, I think it sounds sensible to do as Sofia suggests and add in a high number such as `[^200]` and leave all the other notes as they are. My preference is that we use the Markdown syntax consistently throughout.